scripts/generate-AUTHORS.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2016 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <http://www.mumble.info/LICENSE>.
+
+from __future__ import (unicode_literals, print_function, division)
+
+import sys
+import subprocess
+import codecs
+import locale
+
+blacklist = (
```

Comment:
Thanks for the tip! I think this makes sense.

However, I think we should land with the current script as-is. Then we can port it to use .mailmap in a follow-up.